app/src/main/java/com/github/steroidteam/todolist/viewmodel/TodoListViewModel.java

```diff
@@ -74,6 +75,7 @@ public void renameTask(int index, String newBody) {
     }
 
     public void setTaskDone(int index, boolean isDone) {
+        Log.d("DEBUG", todoListSelected.getValue().toString());
```

Comment:
I'm guessing we don't want this here? ;)

Reply:
Oh yep this one was forgotten thanks ^^